include/maths/time_series/CTimeSeriesModel.h

```diff
@@ -195,6 +195,9 @@ class MATHS_TIME_SERIES_EXPORT CUnivariateTimeSeriesModel : public common::CMode
     //! Unpack the weights in \p weights.
     static TDoubleWeightsAry unpack(const TDouble2VecWeightsAry& weights);
 
+    //! Apply time shift at the time port \p time by \p shift amount of seconds.
```

Comment:
This and others below.
```suggestion
    //! Apply time shift at the time \p time by \p shift amount of seconds.
```